codemod/__init__.py

```diff
@@ -0,0 +1 @@
+from codemod.base import *  # noqa
```

Comment:
@modocache yes it is acronym for "no quality-assurance"

importing * is considered a code smell by PEP8 and Flake will think it is an error so tests will not pass.

adding # noqa makes flake8 and ladscape.io to ignore that line (http://flake8.readthedocs.org/en/latest/)

In that particular case, exposing objects to a top level module, importing in this way is useful.